lib/serializer.ts

```diff
@@ -512,13 +558,46 @@ export class Serializer {
    * @returns {object|string|Array|number|boolean|Date|stream} A valid deserialized Javascript object
    */
   deserialize(mapper: Mapper, responseBody: any, objectName: string): any {
-    if (responseBody === null || responseBody === undefined) return responseBody;
+    if (responseBody == undefined) {
```

Comment:
=== is different than ==

Reply:
and why remove the responseBody === null check?

Reply:
This is an intentional style change based on `foo === undefined || foo === null` being equivalent to `foo == undefined`.

Your input on which style to go with is welcomed--we can configure tslint to only allow `==` in the case of `expr == undefined` comparison.

Reply:
By not having the null check this means that we want to process the responseBody when it is null. Is there are reason to do so?

Reply:
ok learnt something new today. Played in the node REPL.
```
> null == undefined
true
> null === undefined
false
>
```